### DISCUSSION TITLE
[x][closed][manuscript] Experiment study target

### DISCUSSION THREAD

**Discussion by dzerbino** [fairtracks/fairtracks_standard#113] [Team Posts]
I've gone through the ontologies in the standard, and I'm at a loss wrt generalising the concept of an experiment's target. 

When dealing with ChIP-Seq the target is a molecule, and I can refer to a gene and event... however when dealing with say GWAS the 'target' is a disease, and when dealing with segmentation, the target are a bunch of annotation files.... I feel that we are trying to shoehorn too many things into something which mostly makes sense for Blueprint tracks, but not necessarily all of biological experiments.

**Comment by sveinugu** [comment]
@dzerbino I see your point, and thinking from a data representation point of view I agree that the attribute is too wide. 

However, thinking from the perspective of (meta)data integration for downstream tool users, I would argue that this attribute could perhaps be the most important contribution in our standard. What it allows is to seamlessly integrate tracks of completely different types and origins into comparative analyses, which is especially useful for exploratory analyses. One good example of this (not HyperBrowser, to make a better point), is the tool EpiExplorer (https://epiexplorer.mpi-inf.mpg.de/, not updated since 2012, it seems..). The tool basically takes in one or more track files and compares them with a list of widely different tracks from the ENCODE project: https://docs.google.com/spreadsheets/d/1GlNsHrASYHXjEZzln2Iu3o1S1Bha8VlowqKUeSBGke4/pub?output=html&widget=true. In order to label the results, one would need some form of uniform attribute, in this example this would be the "annotation name" column. Using our terms, one could create this column as a combination of target and sample_type, in this case formatted such: "experiment->target (sample->sample_type)". 

I believe that being able to label tracks so easily, independently of type and origin, will be hugely useful!

**Comment by sveinugu** [comment]
Created a new issue on this: https://github.com/fairtracks/fairtracks_standard/issues/11

**Comment by sveinugu** [comment]
Closing this, as there is an issue for it (and a commit from @dzerbino).

**Comment by sveinugu** [comment]
Added [manuscript] and pin
